assets/css/search.css

```diff
@@ -3,7 +3,7 @@
 
 .search .travel-date-input {
-    line-height: 0.125rem;
+    line-height: 1.125rem;
```

Comment:
Same case here as with `archive.css` -- looks like the condition can be just removed.